edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -315,21 +327,21 @@ def _decode_and_verify_token(token, jwt_issuer):
     return decoded_token
 
 
-def _get_signing_jwk_key_set(jwt_issuer, add_symmetric_keys=True):
+def get_signing_jwk_key_set(secret_key, add_symmetric_keys=True, add_asymmetric_keys=True):
     """
     Returns a JWK Keyset containing all active keys that are configured
     for verifying signatures.
     """
     key_set = []
 
-    # asymmetric keys
-    signing_jwk_set = settings.JWT_AUTH.get('JWT_PUBLIC_SIGNING_JWK_SET')
-    if signing_jwk_set:
+    if add_asymmetric_keys:
+        # asymmetric keys
+        signing_jwk_set = settings.JWT_AUTH.get('JWT_PUBLIC_SIGNING_JWK_SET')
```

Comment:
[concern] I think this would be a breaking change in behavior, possibly an unwanted one -- if `JWT_AUTH.JWT_PUBLIC_SIGNING_JWK_SET` is not defined, then from_json would throw.

Reply:
Updated.